controllers/datadogagent/common_rbac.go

```diff
@@ -31,7 +31,7 @@ type roleBindingInfo struct {
 func buildRoleBinding(dda *datadoghqv1alpha1.DatadogAgent, info roleBindingInfo, agentVersion string) *rbacv1.RoleBinding {
 	return &rbacv1.RoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels:    getDefaultLabels(dda, info.name, agentVersion),
+			Labels:    getDefaultLabels(dda, dda.Name, agentVersion),
```

Comment:
Shouldn't these two be `NewPartOfLabelValue(dda).String()` like the rest?

Reply:
Here it is a role not a clusterrole

Reply:
You're right, the namespace of the DDA is not needed in roles, but I wonder if we should just use the same format everywhere for consistency.

`buildAgentClusterAgentRole` (https://github.com/DataDog/datadog-operator/pull/386/files/d695397051a98d999df874dabfeba119ddfde1dd#diff-1ab2275c9afe43600275bda29e5e505bb4ecda385774675935de82f60e27a1b7R1363)  is a role and uses `NewPartOfLabelValue(dda).String()`. That'd need to be changed if we keep `dda.Name` here.

Reply:
good catch